src/main/java/org/prebid/server/handler/admin/SettingsCacheNotificationHandler.java

```diff
@@ -31,10 +35,13 @@ public SettingsCacheNotificationHandler(CacheNotificationListener cacheNotificat
 
     @Override
     public void handle(RoutingContext routingContext) {
-        switch (routingContext.request().method()) {
-            case POST -> doSave(routingContext);
-            case DELETE -> doInvalidate(routingContext);
-            default -> doFail(routingContext);
+        HttpMethod method = routingContext.request().method();
+        if (method.equals(POST)) {
```

Comment:
Was there a reason to stop using switch/case?

Reply:
Yes, previously HttpMethod was an enum, but now HttpMethod is just a class with a bunch of constant instances, and we can't do switch on them :(